lib/src/main/java/uk/gov/di/ipv/cri/kbv/api/strategy/KbvStrategyParser.java

```diff
@@ -0,0 +1,33 @@
+package uk.gov.di.ipv.cri.kbv.api.strategy;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.function.Predicate;
+import java.util.stream.Collectors;
+
+public class KbvStrategyParser {
+    private final String strategy;
+
+    public KbvStrategyParser(String strategy) {
+        this.strategy = strategy;
+    }
+
+    public Strategy parse() {
+        List<Integer> numbers =
+                Arrays.stream(strategy.split("\\D+"))
+                        .filter(Predicate.not(String::isEmpty))
+                        .map(Integer::parseInt)
+                        .collect(Collectors.toList());
+
+        if (numbers.size() != 2) {
+            throw new IllegalArgumentException(String.format("Invalid input string: %s", strategy));
+        }
+        if (numbers.get(0) >= numbers.get(1)) {
```

Comment:
```
int firstNumber = numbers.get(0);
int secondNumber = numbers.get(1);
```
then use these instead of doing `numbers.get(n)` for better readability please

Reply:
Ok